system/iptables_raw.py

```diff
@@ -304,6 +321,12 @@ def _check_compatibility(self):
         else:
             Iptables.module.fail_json(msg="Could not fetch iptables version! Is iptables installed?")
 
+    # Check if /etc/init.d/iptables-persistent for older Debian based systems
+    # Check if /etc/init.d/netfilter-persistent for newer Debian based systems
+        if self.is_debian():
+            if not os.path.isfile('/etc/init.d/netfilter-persistent') and not os.path.isfile('/etc/init.d/iptables-persistent'):
+                Iptables.module.fail_json(msg="Could not detect iptables-persistent package. Is iptables-persistent installed?")
```

Comment:
Because some people are using pre/post scripts as a replacement to `iptables-persistent`, I don't want to force them to have it installed. If you are going to use `iptables-persistent` you are going to have it installed anyway, so it's not an issue to those users.